packages/replay-next/src/contexts/FocusContext.tsx

```diff
@@ -179,14 +186,22 @@ export function FocusContextRoot({ children }: PropsWithChildren<{}>) {
 
   const focusContext = useMemo<FocusContextType>(
     () => ({
+      activeRange,
       enterFocusMode: () => {},
       isTransitionPending,
       rangeForDisplay: range,
       range: deferredRange,
       update: updateFocusRange,
       updateForTimelineImprecise,
     }),
-    [deferredRange, isTransitionPending, range, updateFocusRange, updateForTimelineImprecise]
+    [
+      activeRange,
+      deferredRange,
```

Comment:
I don't think having "active range", "deferred range", and "range" is good. I think particularly, naming something "deferred range" internally but exposing it as "range" is...confusing. Reading this the first time through– even being familiar with the constraints, I was a bit confused.

I know this is kind of subjective but I think we should try to pick clearer names. I propose:
* rangeForDisplay: Only used by the Timeline UI for display purposes
* rangeForSuspense: Used for anything that might _suspend_ based on the value
* range: Used for things that _don't_ need to suspend